p4-help-functions

```diff
@@ -183,3 +188,11 @@ p4-pick-client() {
     echo >&2
     echo ${clients[${result}]}
 }
+
+function p4-guess-username() {
```

Comment:
I'd prefer calling it p4-resolve-username rather than guess - especially since this is the function used by all the other p4 scripts.